styles/antd-theme.less

```
@@ -0,0 +1,3 @@
+@import "../node_modules/antd/dist/antd.less";
+
+@primary-color: #642580;
```

We were using #722ED1 in the figma design that's a bit more light, and I think looks better.

```suggestion
@primary-color: #722ED1;
```